ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Storages.resource

```diff
@@ -114,6 +125,12 @@ Delete Storage
     Workbenches.Click Action From Actions Menu    item_title=${name}    item_type=storage    action=Delete
     Handle Deletion Confirmation Modal    item_title=${name}    item_type=storage    press_cancel=${press_cancel}
 
+Edit Storage
+    [Documentation]    Edits a cluster storage from DS Project details page
+    [Arguments]     ${name}
+    Workbenches.Click Action From Actions Menu    item_title=${name}    item_type=storage    action=Edit
```

Comment:
hey can you try using `ODHDashboard.Click Action From Actions Menu`? I created this to be the common keyword for doing the same thing for all the DS Project components. I didn't replace it in everywhere yet. But this could be used from now on as default option

Reply:
Hi Berto, thank you for the suggestion - looks like it works as expected. Thus I updated it here and also few lines above so it's consistent in this file.